### ISSUE TITLE
[Bug] ContentDescription on Browser Icons

### ISSUE THREAD

**Issue by Sagar0-0** [ankidroid/Anki-Android#15308]
###### Reproduction Steps

1. Go to Card browser
2. Click on search
3. Long press on Back and cross button
4. No content description on them

###### Expected Result
Long press should show description of icons

###### Actual Result
No description is present
![photo_6156653370924973103_y](https://github.com/ankidroid/Anki-Android/assets/85388413/9d8ad69d-e7fe-44dc-837f-bcf6595b5bca)
![photo_6156653370924973104_y](https://github.com/ankidroid/Anki-Android/assets/85388413/57e6333a-2bb3-45b2-ac5c-9fac7db2bdef)

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".
AnkiDroid Version = 2.17alpha14-debug (f034ae348e0f993392e91875d4ad094341f570ea)

Backend Version = 0.1.32-anki23.10.1 (23.10.1 fac9e0ee1436ba5ac3366c72dd9394a6e692b1cf)

Android Version = 13 (SDK 33)

ProductFlavor = amazon

Manufacturer = realme

Model = RMX3371

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; RMX3371 Build/TP1A.220905.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/120.0.6099.43 Mobile Safari/537.36

ACRA UUID = cddde5e0-617c-4fbe-8864-951ba7e89322

Crash Reports Enabled = false

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by Sagar0-0** [comment]
Confirm it and I will start working

**Comment by Sagar0-0** [comment]
This is an Invalid issue, I verified it. We are using android SearchView which is providing these icons and we cannot extend it.

**Comment by Sagar0-0** [comment]
@david-allison I found a programmatical way of adding contentDescription or overriding these SearchView items. Do you think I should reopen this issue?

**Comment by david-allison** [comment]
Sure!

**Comment by Sagar0-0** [comment]
Yes, please go ahead

**Comment by gayatriii0803** [comment]
> I found a programmatical way of adding contentDescription or overriding these SearchView items

@Sagar0-0 Will you please guide me, how to solve this issue

**Comment by jainv4156** [comment]
hey @gayatriii0803 are you working on it  or should i continue on this issue
